### PR TITLE
Correctly determine when a track is new

### LINKED ISSUE #5835: [Tracks] Almost all tracks have the `New` badge
From what I have observed in the Tracks section, the tracks which have 0 concepts have a `New` badge even if they were present in v2 and some tracks which are newly created and have concepts don't show a `New` badge

### PR DESCRIPTION
**Pull request by ErikSchierboom**
Closes https://github.com/exercism/exercism/issues/5835

### CODE REVIEW

File: app/models/track.rb [hunk 1]
```diff
@@ -92,6 +92,10 @@ def average_test_duration
     git.average_test_duration + INFRASTRUCTURE_DURATION_S
   end
 
+  def new?
```
Comment (ErikSchierboom) [inline app/models/track.rb]: Do you think it makes sense to add this to the `Track` model, or should this just reside in `SerializeTrack`?

File: app/models/track.rb [hunk 2]
```diff
@@ -92,6 +92,10 @@ def average_test_duration
...
+    created_at > Time.zone.now - 6.months
```
Comment (ErikSchierboom) [inline app/models/track.rb]: I think we said 6 months, didn't we?

File: app/models/track.rb (hunk 1 above)
Comment (iHiD) [reply app/models/track.rb]: I think do it in SerializeTrack. If we ever then use it somewhere else, we can move it here, but I think keeping the models small is good.

File: app/serializers/serialize_track.rb [hunk 3]
```diff
@@ -18,10 +18,7 @@ def call
       icon_url: track.icon_url,
       tags: map_tags(track.tags),
       last_touched_at: user_track.last_touched_at&.iso8601,
-
-      # TODO: Set these
-      is_new: true,
-
+      is_new: track.created_at > Time.zone.now - 6.months,
```
Comment (iHiD) [inline app/serializers/serialize_track.rb]: I normally use `Time.current` here instead of `Time.zone.now` - probably worth changing this to be consistent.

**Pull request merged**